easybuild/easyconfigs/q/qmflows/qmflows-0.12.1-foss-2021a.eb

```diff
@@ -0,0 +1,94 @@
+easyblock = 'PythonBundle'
+
+name = 'qmflows'
+version = '0.12.1'
+
+homepage = 'https://github.com/SCM-NV/qmflows'
+description = """This library tackles the construction and efficient execution 
+of computational chemistry workflows. This allows computational chemists to use 
+the emerging massively parallel compute environments in an easy manner and focus 
+on interpretation of scientific data rather than on tedious job submission 
+procedures and manual data processing.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = {
+    ('Python', '3.9.5'),
+    ('RDKit', '2021.03.4'),
+    ('HDF5', '1.10.7'),
+    ('SciPy-bundle', '2021.05'),
+    ('Rust', '1.52.1')
+}
+
+use_pip = True
+
+exts_list = [
+    ('more-itertools', '9.0.0', {
+        'checksums': ['5a6257e40878ef0520b1803990e3e22303a41b5714006c32a3fd8304b26ea1ab'],
+    }),
+    ('dill', '0.3.6', {
+        'checksums': ['e5db55f3687856d8fbdab002ed78544e1c4559a130302693d839dfe8f93f2373'],
+    }),
+    ('plams', '1.5.1', {
+        'modulename': 'scm.plams',
+        'checksums': ['5e77632be84a8c12172eb426589cb80187b5b31325d917bdbeb0798f9c5aed46'],
+    }),
+    ('pyparsing', '3.0.9', {
+        'checksums': ['2b020ecf7d21b687f219b71ecad3631f644a47f01403fa1d1036b0c6416d70fb'],
+    }),
+    ('PyYAML', '6.0', {
```

Comment:
There is already an easyconfig for PyYAML, this extension can be replaced by a regular dependency